### PR TITLE
feat(postman): updated edc test to 0.7.x and dtr to 0.4.3 and idp usage

### PR DESCRIPTION
**Pull request by tom-rm-meyer-ISST**
## Description

Updates postman collection for the INT test so that the following changes are reflected:
- edc version bump to 0.7.x
- dtr version bump to 0.4.3
- dtr idp integration

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: local/postman/puris-integration-test.postman_collection.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
+		"_postman_id": "366237ef-086a-48bb-b723-c0da31f8e1ca",
 		"name": "puris-integration-test",
-		"description": "The collection assumes the following roles:\n\n- Customer PURIS (upper tier)\n    \n- Supplier PURIS (lower tier)\n    \n\nIt creates the opposite partner and the respective view on the semiconductor material or product.\n\nTo run the tests correctly, take the environment and fill the respective secrets:\n\n- CUSTOMER_EDC_API_KEY\n- CUSTOMER_PURIS_BACKEND_API_KEY\n- SUPPLIER_EDC_API_KEY\n- SUPPLIER_PURIS_BACKEND_API_KEY\n    \n\nWhen using the another environment, update / create a new environment.",
+		"description": "The collection assumes the following roles:\n\n- Customer PURIS (upper tier)\n- Supplier PURIS (lower tier)\n    \n\nIt creates the opposite partner and the respective view on the semiconductor material or product.\n\nTo run the tests correctly, take the environment and fill the respective secrets:\n\n- CUSTOMER_EDC_API_KEY\n- CUSTOMER_PURIS_BACKEND_API_KEY\n- CUSTOMER__MANAGE_CLIENT_SECRET\n- SUPPLIER_EDC_API_KEY\n- SUPPLIER_MANAGE_CLIENT_SECRET\n    \n\nWhen using the another environment, update / create a new environment.",
```
Comment (ReneSchroederLJ) [inline local/postman/puris-integration-test.postman_collection.json:5]: Does it really make sense that supplier backend api key was removed from this list?

File: local/postman/puris-integration-test.postman_collection.json [hunk 2]
```diff
@@ -3652,6 +3652,62 @@
 				{
 					"name": "Check Customer DTR",
 					"item": [
+						{
+							"name": "CUSTOMER IDP CHECK Copy",
```
Comment (ReneSchroederLJ) [inline local/postman/puris-integration-test.postman_collection.json:3656]: "Copy" should be removed

**Review by ReneSchroederLJ — APPROVED** [review]
Looks good to me. There are only some very minor comments.

File: local/postman/puris-integration-test.postman_collection.json (hunk 2 above)
Comment (tom-rm-meyer-ISST) [reply local/postman/puris-integration-test.postman_collection.json]: Well seen, fixed!

File: local/postman/puris-integration-test.postman_collection.json (hunk 1 above)
Comment (tom-rm-meyer-ISST) [reply local/postman/puris-integration-test.postman_collection.json]: No, you're right. Fixed!

**Review by mhellmeier — APPROVED** [review]

**Pull request merged**